src/js/tour.js

```diff
@@ -408,7 +409,9 @@ export class Tour extends Evented {
    * @private
    */
   _hideModalOverlay() {
-    document.body.classList.remove(modalClassNames.isVisible);
+    if (document.body.classList.contains(modalClassNames.isVisible)) {
+      document.body.classList.remove(modalClassNames.isVisible);
```

Comment:
Do we need the `if` check? Calling remove shouldn't fail if there's no match

Reply:
true, was just overdoing it I suppose